### ISSUE TITLE
Can add Covid free district

### ISSUE THREAD

**Issue by yogesh7401** [covid19india/covid19india.github.io#2516]
**Can add Covid free district list**
It will be use full for user and create relief for district peoples of Covid free districts 

**How to do ? (My Suggestion)**
Some state like telungana there is 0 data and API not feed active cases , you get active cases by subracting deceased and recovered cases from Confirmed cases , so there itself check confirmed cases > 0 and active case === 0 and store the district key and display the result as covid free districts  

**For example**
```js
let covidFreeDistrict = []
if(active === 0 && confirmed > 0) {
    covidFreeDistrict.push({districtName,key})
}
```

**Comment by prablr79** [comment]
I agree with @yogesh7401 , indeed its usefull at pan india level for quick travel or emergency movements

**Comment by prablr79** [comment]
any thoughts on this request ?

**Comment by sayna3311** [comment]
What we can do is, for each state, make a list of covid free districts and render the list on each state's page?

**Comment by B-AYUSHMAN-PRUSTY** [comment]
hey i am interested to add covid free district but i don't know how to do. Can you help me to add district
